.github/actions/sccache/action.yml

```diff
@@ -8,6 +8,8 @@ inputs:
 
 runs:
   using: composite
+  env:
```

Comment:
I don't think this applies the env var to when sccache is actually running in the `run` steps does it?